cmd/scw/testdata/test-all-usage-baremetal-server-install-usage.stderr.golden

```diff
@@ -3,6 +3,10 @@ Install an OS on the server associated with the given ID.
 USAGE:
   scw baremetal server install <server-id> [arg=value ...]
 
+EXAMPLES:
+  Install an OS on a given server
+    scw baremetal server install
```

Comment:
The example is not working

Reply:
Will be fixed by the scaleway-bot with generated code